pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+# PHEP Editor Responsibilities & Workflow
+<a name="phep-editor-responsibilities-workflow"></a>
+A PHEP editor must be added to the `@heliophysicsPy/pep-editors` group on GitHub and must watch the `standards` repository.
+
+Other community members with write access to the `standards` repository may handle the tasks that would normally be taken care of by the PHEP editors. Alternately, even those with write access may request assistance from PHEP editors by mentioning `@heliophysicsPy/phep-editors` on GitHub.
+
+For each new PHEP that comes in an editor does the following:
+
+* Read the PHEP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to be accepted.
+* The title should accurately describe the content.
+* The file name extension is correct (i.e. `.md`).
+* Ensure that everyone listed as a co-author of the PHEP who has write access to the PHEP repository is added to [.github/CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+* Skim the PHEP for obvious defects in language (spelling, grammar, sentence structure, etc.). Editors may correct problems themselves, but are not required to do so.
+* If a project is portrayed as benefiting from or supporting the PHEP, make sure there is some direct indication from the project included to make the support clear. This is to avoid a PHEP accidentally portraying a project as supporting a PHEP when in fact the support is based on conjecture.
+
+If the PHEP isn't ready, an editor will send it back to the author for revision, with specific instructions. If Markdown formatting is a problem, ask the author(s) to resubmit.
+
+Once the PHEP is ready for discussion, a PHEP editor will:
+
+* Assign a PHEP number (almost always just the next available number, generally avoiding special/joke numbers).
+* Check that the author has correctly labeled the PHEP's type ("Standards Track", "Informational", or "Process"), and marked its status as "Draft".
+* Mark the PR ready for review.
+* Inform the author of the next steps (update the PHEP with the PR link, post an announcement, etc).
+
+Updates to existing PHEPs should be submitted as comments in the GitHub pull request.
+
+PHEP editors don't pass judgment on PHEPs. They merely do the administrative & editorial part (which is generally a low volume task). They may participate in the community discussion of a PHEP but have no obligation to be advocates.
+
+Disagreements with editors must be mediated by PyHC leadership (as documented in [PHEP audience](#phep-audience)) and they may impose sanctions, including removal of the editor role.
+
+# PHEP 1 Approval
+
+This PHEP is subject to the same review and approval process that it proposes.
+
+# Copyright
+This document is placed in the public domain or under the CC0-1.0-Universal license, whichever is more permissive.
```

Comment:
Suggestion for clarity: `This document is EITHER placed in the public domain or under the CC0-1.0-Universal license, whichever is more permissive.`

Reply:
My inclination would be to use CC BY 4.0, which adds the requirement of attribution.  Attribution can be a really helpful practice since it gives end readers information about the provenance of ideas, and where they can go to get more information.

Reply:
Another case where the starting point is PEP-1 verbatim.

NASA is strongly pushing CC0. They're emphasizing the distinction between the _legal requirement_ of a license and the _ethical norm_ of citation. If I can legitimately dig up the open science 101 slides I'll link them here (not sure if they're out yet).

So we could add, even in the copyright section, "In accordance with community norms, users should cite this PHEP as..." and we could include number, title, authors, DOI...whatever seems appropriate.

[CC is fairly unhelpful about PD/CC0 dual licensing](https://creativecommons.org/faq/#how-do-cc-licenses-operate) probably because [they don't think it's necessary](https://wiki.creativecommons.org/wiki/CC0_FAQ#Does_CC0_really_eliminate_all_copyright_and_related_rights.2C_everywhere.3F). I *think* if we dual-license it probably should be AND. That leaves out any argument over "which is more permissive" and it's just plain available to anyone. My memory (from quite some time back!) is that CC0 came about because in some jurisdictions it wasn't clear that it was possible to put something into the public domain [[e.g](https://opensource.stackexchange.com/questions/9871/why-is-there-no-public-domain-licensing-in-europe)]. So CC0 was an out.

Temptation here is to say "PSF's lawyers looked at this, let's use their wording unless we want other lawyers to look at it".

Reply:
@jtniehof has clearly thought about this license issue deeper than I have, so I'll differ to his best judgments. I buy the reasoning.

Reply:
I have added citation information which hopefully nudges people towards using it (and it'll be in the template): https://github.com/jtniehof/standards/blob/0cd4f81ca4dd6b498606aadb3bbe451bd0031b9b/pheps/phep-0001.md?plain=1#L322